moveit_ros/benchmarks/examples/demo_obstacles.yaml

```diff
@@ -0,0 +1,33 @@
+# This is an example configuration that loads the "Kitchen" scene from the
+# local MoveIt warehouse and benchmarks the "manipulator" group over all pairs
+# of motion plan queries and start states in the Kitchen scene.
+
+# Five planners from two different plugins are run a total of 50 times each, with a
+# maximum of 10 seconds per run.  Output is stored in the /home/benchmarks directory.
```

Comment:
> Output is stored in the /home/benchmarks directory.
     
This is not true, right? Line 18 has a different location

Reply:
yes, you are right, fixed!